docs/developers/nft-tokens.md

```diff
@@ -586,13 +643,13 @@ Fill all the attributes as you think.
     Value: 0
     GasLimit: 60000000
     Data: "ESDTNFTCreate" +
-          "@414c432d317132773365" +   # previously fetched token identifier
-          "@" + <initial quantity in hexadecimal encoding> +
-          "@" + <NFT name in hexadecimal econding> +
-          "@" + <Royalties in hexadecimal encoding> +
-          "@" + <Hash in hexadecimal encoding> +
-          "@" + <Attributes in hexadecimal encoding> +
-          "@" + <URI in hexadecimal encoding> +
+          "@414c432d317132773365" +  # previously fetched token identifier
+          "@01" + # quantity: 1
+          "@42656175746966756c20736f6e67" + # NFT name: 'Beautiful song' in hexadecimal encoding
+          "@1d4c" + # Royalties: 7500 =75%c in hexadecimal encoding
+          "@00" + # Hash: 00 in hexadecimal encoding
+          "@6d657461646174613a697066734349442f736f6e672e6a736f6e3b746167733a736f6e672c62656175746966756c2c6d75736963" + # Attributes: metadata:ipfsCID/song.json;tags:song,beautiful,music	 in hexadecimal encoding> +
+          "@55524c5f746f5f646563656e7472616c697a65645f73746f726167652f736f6e672e6d7033" + # URI: URL_to_decentralized_storage/song.mp3 in hexadecimal encoding> +
           "@" + <URI in hexadecimal encoding> +
```

Comment:
_perhaps we can specify that this is_ additional optional URI in hexadecimal encoding